src/PhpPact/Consumer/Model/ConsumerRequest.php

```diff
@@ -154,20 +167,15 @@ public function setQuery(string $query): self
      */
     public function addQueryParameter(string $key, string $value): self
```

Comment:
`pactffi_with_query_parameter_v2` can support multiple values and matching rules. But I implement it incorrect. So I think it's better to move it to other pull request.

https://github.com/pact-foundation/pact-reference/blob/master/rust/pact_ffi/src/mock_server/handles.rs#L679

Reply:
I added a commit to this PR anyway. See [bb256ef](https://github.com/pact-foundation/pact-php/pull/292/commits/bb256ef552c8d98860d601002a5382563130f237)